packages/region-provider/src/defaultProvider.ts

```diff
@@ -8,10 +8,7 @@ import { Provider } from "@aws-sdk/types";
 
 export type RegionProviderConfiguration = EnvConfiguration & SharedConfigInit;
 
-export function defaultProvider(
+export const defaultProvider = (
   configuration: RegionProviderConfiguration = {}
-): Provider<string> {
-  return memoize(
```

Comment:
QQ: Why did the formatting change here?

Reply:
It's now an arrow function.
We're slowly switching to arrow functions for no binding of `this` https://www.freecodecamp.org/news/3d851d7f0b26/